cargo-shuttle/src/lib.rs

```diff
@@ -217,8 +217,14 @@ impl Shuttle {
         };
 
         // 5. Initialize locally
-        init::cargo_init(path.clone(), project_args.name.clone().unwrap())?;
-        init::cargo_shuttle_init(path.clone(), framework)?;
```

Comment:
@paulotten This `cargo_shuttle_init` used to do the project name change. You need to restore that part of it.